configure.ac

```diff
@@ -674,6 +722,23 @@ echo "Event handlers:"
 AM_COND_IF([ENABLE_SAMPLEEVH],
 	[echo "    Sample event handler:  yes"],
 	[echo "    Sample event handler:  no"])
+AM_COND_IF([ENABLE_JAVASCRIPT_MODULES], [
+	echo "Using npm:                 $NPM"
```

Comment:
Please put this indented in the JavaScript modules thingie: I don't want people to think you need npm to use Janus.